### DISCUSSION TITLE
Pin group when scrolling

### DISCUSSION THREAD

**Discussion by emubaybiodomes** [TT-ReBORN/Georgia-ReBORN#150] [General]
Is there an option to "pin when scrolling" in the playlist?

With SimPlaylist this option makes the top release header sticky, rather than letting it "naturally" disappear as you scroll down the playlist. 

If that description isn't clear, what I'm talking about is this:

https://github.com/TT-ReBORN/Georgia-ReBORN/assets/146593097/1ac89908-581a-4207-b20a-779f986a1c9b

**Comment by TT-ReBORN** [comment]
Hi @emubaybiodomes,

the playlist does not have such option, but I can imagine to have this feature in a online web player...

-TT

**Reply by emubaybiodomes** [reply]
Thanks your response. I'm not totally sure I know what you mean though. Why an online web player?

**Reply by TT-ReBORN** [reply]
Well this feature is typically used in web design for example in the top menu navigation. 
I'm not fond of to implement and use it in the playlist besides I have a lot of other important work to finish first.

**Reply by emubaybiodomes** [reply]
Aah OK, I get you now. 

I find the option very useful especially when dealing with releases like box sets, anthologies, comps or other releases that have many 10s of tracks, as it can save a lot of scrolling up! Apparently I am not the only one [https://hydrogenaud.io/index.php/topic,123009.msg1016094.html](url) but it's quite possible we're in a tiny minority. Suffice to say, if  this option was ever implemented then it'd be massively appreciated. 

As a total aside thanks so much for your work on this theme. After many years I finally feel like this is the theme that motivates me to leave my old setup behind. There are a few things I can't yet work out how to do yet but I'll probably get there!

**Reply by TT-ReBORN** [reply]
You can increase the playlist scroll steps via top menu `Options` > `Player controls` > `Playlist` > Mouse wheel scroll steps
You can do this for all scrollbars in the theme and If you put a high value you can scroll much faster ;-)
